frontend/src/metabase-lib/test-helpers.ts

```diff
@@ -42,3 +42,21 @@ export function createQuery({
   const metadataProvider = createMetadataProvider({ databaseId, metadata });
   return ML.fromLegacyQuery(databaseId, metadataProvider, query);
 }
+
+export const findColumn =
+  (query: ML.Query, columns: ML.ColumnMetadata[]) =>
+  (tableName: string, columnName: string): ML.ColumnMetadata => {
+    const column = columns.find(column => {
+      const displayInfo = ML.displayInfo(query, column);
+      return (
+        displayInfo?.table?.name === tableName &&
+        displayInfo?.name === columnName
+      );
+    });
+
+    if (!column) {
+      throw new Error(`Could not find ${tableName}.${columnName}`);
+    }
+
+    return column;
+  };
```

Comment:
Extracting a helper that was implemented a few times in different MLv2 test suites. Not sure if curring here makes things better or actually makes things more confusing, would appreciate your thoughts

Reply:
In the BE we would call such a function sg. like `column-finder`, I think.